### PR TITLE
feat:input documentation

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/153274535-24c62284-2f53-4215-a0f1-dcdd3c1fb2c1.mp4

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.input-container {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
```
Comment (dikshadhote) [inline components/input/input.css:4]: Use rem for responsiveness and px for values below 10px

File: components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    font-size: var(--size-1);
+}
+
+label {
+    display: block;
+}
+
+.input {
+    border: 1px solid var(--text-secondry);
+    height: 35px;
```
Comment (dikshadhote) [inline components/input/input.css:17]: same as above

File: components/input/input.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+p {
+    font-size: var(--size-1);
+}
+
+label {
+    display: block;
+}
+
+.input {
+    border: 1px solid var(--text-secondry);
```
Comment (dikshadhote) [inline components/input/input.css:16]: text-secondary doesn't give idea about color, use proper naming convention for eg : text-secondary-grey

File: components/input/input.css [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    border: 1.5px solid var(--complementary-green-color);
+}
+
+.success-msg {
+    color: var(--complementary-green-color);
+}
+
+i {
+    padding: 10px;
+}
```
Comment (dikshadhote) [inline components/input/input.css:45]: Make separate utlity class for padding, instead of giving to element

File: components/input/input.css [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+.icon {
+    position: relative;
+    right: 50px;
```
Comment (dikshadhote) [inline components/input/input.css:49]: same as above (rem)

**Review by dikshadhote — CHANGES_REQUESTED** [review]
Use proper naming conventions and units

**Review by devdeadviz — COMMENTED** [review]
Diksha has already added suggested changes, have a look at it.

**Review by samyakshah3008 — APPROVED** [review]
Looks good to merge Stuti!

**Pull request merged**